docs/examples/config/lp_paleobiology.rst

```diff
@@ -159,11 +159,11 @@ The output should appear similar to::
 After this has finished, our output will be in our ``[dataset_dir]`` (using the provided config, this will be ``dataset/``.
 
 Here are the files that were created in training:
-Let's check again what was added in the ``datasets/custom_lp_example/`` directory. For clarity, we only list the files that were created in training. Notice that several files have been created, including the trained model, the embedding table, a full configuration file, and output logs:
```

Comment:
Is removing custom_lp_example supposed to be a part of this PR?

Reply:
So this PR includes the documentation changes from #89. Bad practice to include that here. Can keep or remove.